README.md

```diff
@@ -222,9 +205,19 @@ The primary caveat is that if a `login_url` is supplied in `setStatus`, it could
 
 ## Interaction with other FedCM Features
 
+The FedCM features addressed here should not be considered exhaustive; instead, this outlines features where special consideration is
+required. Unless otherwise stated, any FedCM functionality should be available to IdPs taking advantage of Accounts Push, Optional Endpoints, or Config Push. 
+
+Features that should work with no modification include:
+
+* Multiple identity providers in one `navigator.credentials.get()` call.
+* Login, Domain, and Account Hints for accounts filtering.
+* Fields and params for identity assertion retrieval.
+* `IdentityProvider.getUserInfo()` (in an iframe with an origin matching the IdP origin)
+
 ### "Login-Status" Headers
 
-An IdP sending `Login-Status: logged-in` refreshes the expiration timer on the stored profile information by resetting the browser's last-modified timestamp for that site's Login Status information.
+An IdP sending the `Login-Status: logged-in` response header refreshes the expiration timer on the stored profile information by resetting the browser's last-modified timestamp for that site's Login Status information.
```

Comment:
Below, but farther than GitHub will let me comment: The n.l.setStatus example has indentation issues.